tests/conftest.py

```diff
@@ -13,6 +13,7 @@
 def notify_function_process():
     cwd = dirname(dirname(__file__))
 
+    # pylint: disable=consider-using-with
```

Comment:
This has been ignored because yielding within the context manager doesn't seem to work with pytest. It boots the service fine, but it does not tear down.